openedx/core/djangoapps/credit/signals.py

```diff
@@ -5,15 +5,79 @@
 
 import logging
 
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
 from django.dispatch import receiver
 from django.utils import timezone
 from opaque_keys.edx.keys import CourseKey
-
+from openedx_events.learning.signals import (
+    EXAM_ATTEMPT_ERRORED,
+    EXAM_ATTEMPT_REJECTED,
+    EXAM_ATTEMPT_RESET,
+    EXAM_ATTEMPT_SUBMITTED,
+    EXAM_ATTEMPT_VERIFIED
+)
+
+from openedx.core.djangoapps.credit.api.eligibility import (
+    is_credit_course,
+    remove_credit_requirement_status,
+    set_credit_requirement_status
+)
 from openedx.core.djangoapps.signals.signals import COURSE_GRADE_CHANGED
 
+User = get_user_model()
+
 log = logging.getLogger(__name__)
 
 
+def handle_exam_event(signal, event_data, credit_status=None):
```

Comment:
Should the consumer/receiver code be in a `handlers.py` file as to match the pattern of the other consumer PRs?

Reply:
yeah this app follows an older pattern of just putting _everything_ in signals.py. But I'm noticing this is just handlers so should be too disruptive to refactor. I'll take a stab at it